docs/binary.md

```diff
@@ -639,6 +640,21 @@ When interacting with the XML format, care must be taken because `UniqueId` is s
 
 When an array of `UniqueId` values is present, the bytes are subject to [byte interleaving](#byte-interleaving).
 
+### Font
+**Type ID `0x20`**
+
+The `Font` type is a struct composed of two `String` values, a `u8`, and a `u16`:
+
+| Field Name   | Format              | Value                           |
+| :----------- | :------------------ | :------------------------------ |
+| Family       | [`String`](#string) | The font family content url     |
+| Weight       | `u16`               | The weight of the font          |
+| Style        | `u8`                | The style of the font           |
+| CachedFaceId | [`String`](#string) | The cached content url of the ttf file |
+
+The `Weight` and `Style` fields are stored as little-endian unsigned integers. These are usually treated like enums, and to assign them in Roblox Studio an Enum is used. Interestingly, the `Weight` is _always_ stored as a number in binary and xml, but `Style` is stored as a number in binary and as text in xml.
```

Comment:
```suggestion
The `Weight` and `Style` fields are stored as little-endian unsigned integers. These are usually treated like enums, and to assign them in Roblox Studio an Enum is used. Interestingly, the `Weight` is *always* stored as a number in binary and XML, but `Style` is stored as a number in binary and as text in XML.
```

By convention, we use `*` for italics in spec files. Also as previously stated, we capitalize acronyms.